src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -5,19 +5,17 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ImageHandler : ViewHandler<IImage, FrameworkElement>
+	public partial class ImageHandler : ViewHandler<IImage, Image>
 	{
 		double _lastScale = 0.0;
 
-		protected Image? RealNativeView { get; set; }
+		protected override Image CreateNativeView() => new Image();
 
-		protected override FrameworkElement CreateNativeView()
-		{
-			RealNativeView = new Image();
-			return new Border { Child = RealNativeView };
-		}
+		public override bool NeedsContainer =>
+			VirtualView?.BackgroundColor != null ||
+			base.NeedsContainer;
```

Comment:
I also have to override the `NeedsContainer` property to control whether or not the handler is using a container or not.